GradedModulePresentationsForCAP/examples/CohP1.g

```diff
@@ -15,7 +15,7 @@ S := GradedRing( Q["x,y"] );
 Sgrmod := GradedLeftPresentations( S );
 #! The category of graded left f.p. modules over Q[x,y] (with weights [ 1, 1 ])
 InfoOfInstalledOperationsOfCategory( Sgrmod );
-#! 40 primitive operations were used to derive 232 operations for this category which
+#! 40 primitive operations were used to derive 246 operations for this category which
```

Comment:
I think these numbers should not change, because they have nothing to do with `Opposite` as far as I can tell. Do you have these numbers from the CI or from your local computer? In the latter case, probably `Toposes` increases this number.

Reply:
Yes, right. I keep forgetting that I need to test `CAP_project` without `Toposes`.

Reply:
... `HigherHomologicalAlgebra`.